### DISCUSSION TITLE
Fourth (Fifth) Account Type: Accounts Receivable

### DISCUSSION THREAD

**Discussion by cachho** [firefly-iii/firefly-iii#8610] [General]
Hey, great tool first of all. I've been using it for a while.

I'm wondering, like we have expenses and revenue, shouldn't we also have the opposite of liabilities, **accounts receivable**?

For the cases where I've sent an invoice and I'm waiting for money, or in a more private setting, that you borrowed money to one of your friends.

I get that most people have more liabilities in their live, than accounts that owe them money, so maybe I'm trying to use this tool too "professionally".

**Comment by JC5** [comment]
There's a drop-down when you create a new liability. Have you seen it? I know it's not the right accounting terminology.
